.github/workflows/mark-prs-stale.yml

```diff
@@ -0,0 +1,21 @@
+# reference: https://github.com/actions/stale
+name: 'Handle stale PRs'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 2 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
```

Comment:
Does the action need writing permission for issues?

Reply:
it didn't, nice catch